wp-content/themes/workingnyc/views/employer-programs/single.twig

```diff
@@ -62,6 +62,52 @@
       {% include 'employer-programs/employer-program.twig' with { post: program } %}
       </div>
       {% endfor %}
+          <div class="carousel-container">
```

Comment:
Please have these cards be the program cards, not placeholders

Reply:
its just for testing purpose I put outside of the for loop as static. once these carousal works then I will move tis insider for loop.